elastic/kibana
# [ResponseOps][Flapping] add flapping state to alert context for action parameters

Resolves https://github.com/elastic/kibana/issues/146613

## Summary
Makes flapping indicator for an alert available in the context variables for action mustache templates under `alert.flapping`
Let me know if we want to change this to be something else

### Checklist


### To verify

- Create a new rule with a connector
- Add the new field Add `{{alert.flapping}}` to the your alert message, and verify that it is in the alert output.
